README.md

```diff
@@ -425,6 +425,15 @@ task_definition:
       cpu_shares: integer
       mem_limit: string
       mem_reservation: string
+  docker_volumes:
+    - name: string
+      scope: string ("shared" or "task")
+      autoprovision: boolean (only valid if scope = "shared")
+      driver: string
```

Comment:
Is there a discrete list of valid entries for this field (or others) and we can call out with a comment? Thinking about how we [annotated task placement](https://github.com/aws/amazon-ecs-cli/pull/586/files) args.

Reply:
There are a bunch of volume plugins out there (aka volume 'drivers'), and new ones will be created in the future; and putting a list here would be misleading because ECS accepts any string for this value.

Reply:
Makes sense. Thanks!